### PR TITLE
Cleanup the JDBC test framework code

### PR DESCRIPTION
**Pull request by thephantomthief**
Signed-off-by: Sharu Goel <goelshar@amazon.com>

### Description

Clean up the JDCB test framework code and remove obsolete framework functionalities
 
### Issues Resolved

N/A

### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is under the terms of the Apache 2.0 and PostgreSQL licenses, and grant any person obtaining a copy of the contribution permission to relicense all or a portion of my contribution to the PostgreSQL License solely to contribute all or a portion of my contribution to the PostgreSQL open source project.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by kuntalghosh — DISMISSED** [review]

File: test/JDBC/src/main/java/com/sqlsamples/Config.java [hunk 1]
```diff
@@ -68,26 +64,15 @@ static Properties readConfig() {
...
-        if(compareWithFile){
-            //disable logging if we are comparing against expected output
-            logger.setLevel(Level.OFF);
-        } else {
-            logger.setLevel(Level.DEBUG);
-            PatternLayout pattern = new PatternLayout("%m%n");
-            FileAppender fileAppender = new FileAppender(pattern, logFileName+ ".log");
-            logger.addAppender(fileAppender);
-
-            if(printLogsToConsole){
```
Comment (congoamz) [inline test/JDBC/src/main/java/com/sqlsamples/Config.java:83]: why are we removing the printLogsToConsole block here?

File: test/JDBC/src/main/java/com/sqlsamples/JDBCStatement.java [hunk 2]
```diff
@@ -9,234 +9,45 @@
...
+            CompareResults.processResults(stmt_bbl, bw, resultsProcessed, resultSetExist, logger);
         } catch (IOException ioe) {
-            ioe.printStackTrace();
+            logger.error("IO Exception: " + ioe.getMessage(), ioe);
         }
-
-        return true;
     }
-}
+}
```
Comment (congoamz) [inline test/JDBC/src/main/java/com/sqlsamples/JDBCStatement.java]: nit: no newline

File: test/JDBC/src/main/resources/junit-platform.properties [hunk 3]
```diff
@@ -0,0 +1,2 @@
+junit.jupiter.execution.parallel.enabled = false
+junit.jupiter.execution.parallel.mode.default = concurrent
```
Comment (congoamz) [inline test/JDBC/src/main/resources/junit-platform.properties]: nit: no newline

File: test/JDBC/src/test/java/com/sqlsamples/TestQueryFile.java [hunk 4]
```diff
@@ -315,82 +309,47 @@ public void TestQueryBatch(String inputFileName) throws SQLException, ClassNotFo
...
+        try {
+            Assertions.assertTrue(result);
+        } catch (AssertionError e) {
+            Throwable throwable = new Throwable(inputFileName + " FAILED! Output diff can be found in '" + diffFile.getAbsolutePath() + "'");
+            throwable.setStackTrace(new StackTraceElement[0]);
+            throw throwable;
         }
     }
-}
+}
```
Comment (congoamz) [inline test/JDBC/src/test/java/com/sqlsamples/TestQueryFile.java]: nit: no newline

File: test/JDBC/src/main/java/com/sqlsamples/Config.java (hunk 1 above)
Comment (thephantomthief) [reply test/JDBC/src/main/java/com/sqlsamples/Config.java:83]: Earlier the framework had two modes of execution but now there is only one (file based comparison only) so the printLogsToConsole flag is only there to print the framework diff now (output the contents of .diff file to the console). The configureLogger function configures logger to dump all exception stack-traces (IOException, ParseException etc.) except for SQLException into a .log file, so the flag is unrelated to this function now.

File: test/JDBC/src/main/java/com/sqlsamples/Config.java [hunk 5]
```diff
@@ -109,28 +94,21 @@ static String createPostgreSQLConnectionString(String URL, String port, String d
     
     private static String constructConnectionString(String prefix) {
```
Comment (thephantomthief) [inline test/JDBC/src/main/java/com/sqlsamples/Config.java]: @congoamz deleted your comment by mistake. I agree we can remove the prefix argument. Will address this in my revision

**Review by congoamz — APPROVED** [review]

**Review by kuntalghosh — APPROVED** [review]

**Pull request merged**